### PR TITLE
3.4 - Remove more deprecated methods usage

### PR DESCRIPTION
**Pull request by antograssiot**

### CODE REVIEW

File: src/View/StringTemplate.php [hunk 1]
```diff
@@ -29,7 +29,7 @@ class StringTemplate
 {
 
     use InstanceConfigTrait {
-        config as get;
+        getConfig as get;
```
Comment (antograssiot) [inline src/View/StringTemplate.php]: I'made not quite sure about this one, is it OK ?

Comment (dereuromark) [inline src/View/StringTemplate.php]: We did the same for Log and alike afaik. Writing here was never supposed to be done for "get".

Comment (antograssiot) [inline src/View/StringTemplate.php]: That what I thought but wanted to draw attention on it to be sure.

Comment (markstory) [inline src/View/StringTemplate.php]: Looks ok to me.

**Pull request merged**